src/main/java/org/broadinstitute/hellbender/engine/VariantWalkerBase.java

```diff
@@ -95,6 +94,15 @@ public final SAMSequenceDictionary getBestAvailableSequenceDictionary() {
     protected abstract Spliterator<VariantContext> getSpliteratorForDrivingVariants();
 
     /**
+     * When performing a query on the primary variant input how many overlapping records should be cached when querying.
```

Comment:
It's not how many records -- it's how many additional **bases** should be queried and cached beyond the current variant upon a cache miss.